fuzzing/disco_fuzzer.go

```diff
@@ -0,0 +1,15 @@
+// Copyright (c) 2021 Tailscale Inc & AUTHORS All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package fuzzing
```

Comment:
I'd rather this live in the same package as the code that it is fuzzing. You can protect this file with a go-fuzz build tag. Ditto for the other new file.